spring-pulsar/src/main/java/org/springframework/pulsar/aot/PulsarRuntimeHints.java

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright 2022 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.pulsar.aot;
+
+import java.util.HashSet;
+import java.util.TreeMap;
+import java.util.stream.Stream;
+
+import org.apache.pulsar.client.admin.internal.OffloadProcessStatusImpl;
```

Comment:
I wonder if we should use the "string" version of the API for any Pulsar "internal" classes?

Reply:
We could do that, but what benefit it brings? If Pulsar client moves things around, we have to change in either case (string or type).

Reply:
Yeh, I guess if they change things we will know quicker as the build will break on compile. this may actually be better than not knowing about it for N time period.